recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.4.0rc1" %}
+{% set org = "cyclus" %}
+{% set branch = "develop" %}
```

Comment:
conda-build does not allow `'-'` in the version number.  That is what I had tried originally.